### DISCUSSION TITLE
Spaced Repetition Plugin does not allow review of flash cards

### DISCUSSION THREAD

**Discussion by Alebron23** [st3v3nmw/obsidian-spaced-repetition#1190] [Q&A] [answered]
Hi, I am new to Obsidian. I have downloaded the spaced repetition plugin and I am using it for flash cards. When I loop through my deck of flash cards, the plugin does not allow me to re-review the cards. If I add a new card to the deck, the plugin will allow me to review that card, but after the flashcard pane closes, it will not allow me to review the deck another time. Any pointers?

https://github.com/user-attachments/assets/6b377556-6eab-49d6-9123-2300e736977b

**Comment by tarakanbg** [comment]
I concur. I've been having the same issue with inline reversed flashcards for a while, may be since the last couple of upgrades and have filed a bug report: https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/1192

**Comment by Alebron23** [comment]
@ronzulu @st3v3nmw Any thoughts?

**Reply by Gusatvoschaivon** [reply]
Having the same problem one year later lol, it doesn't let me do again and doesn't let me have "cards due"

**Comment by dkosarevmusic** [comment, marked as answer]
That's not a bug. This is how the algorithm works. You should use the "Spaced Repetition: Select a deck to cram" command. It ignores review stamps. There's also a similar command for notes.

**Comment by KyleKlus** [comment]
Hi so there was a bug which made some cards not appear, which should be fixed in the next version.

Else the correct answer is this one 😇:
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/1190#discussioncomment-12408866
